assets/airlines/swn.json

```diff
@@ -0,0 +1,17 @@
+{
+    "icao": "swn",
+    "name": "West Air Sweden",
+    "callsign": {
+        "name": "Air Sweden",
+        "callsignFormats": [
+            "####"
+        ]
+    },
+    "fleets": {
+        "default": [
+            ["B762", 4],
```

Comment:
Is it okay to use a 767-200 instead of an 767-200SF, which is the specific variant used?

Reply:
West Air Sweden retired their 767s (https://www.scramble.nl/civil-news/west-air-sweden-retires-b767fs)

Reply:
Removed!